gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -182,6 +182,7 @@ public void close()
         cleanUpWorkingDirectory();
       } finally {
         super.close();
+        fs.close();
```

Comment:
I think it's generally a good practice to close in order to release any resource that object holds. As it now has newInstance of FileSystem, instead of shared one, I think it makes sense to close it to release resource.